chef/cookbooks/network/templates/default/suse-cfg.erb

```diff
@@ -38,8 +38,14 @@ BONDING_MODULE_OPTS=<%=quote("mode=#{@nic.mode} miimon=#{@nic.miimon}")%>
 <% iface["slaves"].each_with_index do |slave,i| -%>
 BONDING_SLAVE_<%=i%>=<%=quote(slave)%>
 <% end -%>
+  <% if iface["mtu"] == 9000 -%>
```

Comment:
The problem (https://www.novell.com/support/kb/doc.php?id=7014947) occurs only on jumbo frames. That means we need this setting only when we activated jumbo frames. I'm not sure if this  setting has also an effect if the mtu not 9000.